.github/workflows/main.yaml

```diff
@@ -0,0 +1,119 @@
+name: Build kernel images
+
+on: 
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  build-python-images:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # version: ["3.8.8", "3.9.13", "3.10.5"] 3.10.5 fails with dependency conflicts
+        # We may need to have separate requirements.txt for each version, or replace
+        # dependencies on the fly
+        version: ["3.8.8", "3.9.13"]
```

Comment:
```suggestion
        version: ["3.8.14", "3.9.14"]
```

Reply:
This is for the jupyterhub datascience-notebook image. I don't see a `.14` version for either: https://hub.docker.com/r/jupyter/datascience-notebook/tags?page=1&name=python-3.9. But I have updated to the latest.